tests/update/patch_before_migration.pm

```diff
@@ -53,7 +53,7 @@ sub patching_sle {
     if (get_var('FULL_UPDATE')) {
         fully_patch_system();
         type_string "reboot\n";
-        $self->wait_boot(textmode => !is_desktop_installed(), ready_time => 600);
+        $self->wait_boot(textmode => !is_desktop_installed(), ready_time => 600, boot_time => 200);
```

Comment:
The change here actually doesn't work, following info showed in your job log: http://10.67.17.147/tests/2032/file/autoinst-log.txt
[2018-01-19T18:06:57.0307 CST] [debug] /var/lib/openqa/share/tests/sle/tests/update/patch_before_migration.pm:56 called opensusebasetest::wait_boot
[2018-01-19T18:06:57.0307 CST] [debug] <<< testapi::check_screen(mustmatch=[
  'grub2'
], timeout=100)

Please refer to function definition in lib/opensusebasetest.pm:
  wait_boot([bootloader_time => $bootloader_time] [, textmode => $textmode] [,ready_time => $ready_time] [,in_grub => $in_grub] [, nologin => $nologin);

There is no "boot_time" parameter defined for wait_boot, it should be "bootloader_time".